DataConnectors/Forcepoint CASB.json

```diff
@@ -0,0 +1,108 @@
+{
+    "id": "Forcepoint",
+    "title": "Forcepoint CASB",
+    "publisher": "Forcepoint CASB",
+    "descriptionMarkdown": "The Forcepoint CASB connector allows you to automatically export CASB logs and events into Azure Sentinel in real-time. This enriches visibility into user activities across locations and cloud applications, enables further correlation with data from Azure workloads and other feeds, and improves monitoring capability with Workbooks inside Azure Sentinel.​",
```

Comment:
1. Please add the full name of the connector in the description as 'CASB (Cloud ..)' - that way the full name is mentioned once at least
2. I see formatting issues when I load this connector at the end of the description 'Azure Sentinel.' Just delete some part of 'Sentinel.' and the retype it to remove rogue characters and import the file to validate using the import test tool/functionality.